validator/allowed_resource_types_test.rego

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2018 Google LLC
```

Comment:
can we update this to 2019?

Reply:
done - I also fixed the sample file name